services/cinema_hall.py

```diff
@@ -0,0 +1,15 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(
+    hall_name: str, hall_rows: int, hall_seats_in_row: int
```

Comment:
Place all arguments with type annotations on a new line to improve the readability.